app/source/css/components/TilesList.styl

```diff
@@ -34,6 +34,7 @@
     &.TilesList--openEnd {
         > *:last-child::after {
             border-bottom: none !important;
+            border-right: none !important;
```

Comment:
Why this is needed? (the default style for TilesList has not an ":after border-right")